cli/azd/docs/external-authentication.md

```diff
@@ -48,7 +48,7 @@ The server should take this request and fetch a token using the given configuati
 
 ```jsonc
 {
-  "result": "success",
+  "status": "success",
```

Comment:
This is updating the documentation to match the implementation (which both VS Code and `azd` already shipped)